types/jquery/index.d.ts

```diff
@@ -146,21 +146,32 @@ interface JQueryStatic {
      * Return a collection of matched elements either found in the DOM based on passed argument(s) or created
      * by passing an HTML string.
      *
-     * @param element_elementArray A DOM element to wrap in a jQuery object.
-     *                             An array containing a set of DOM elements to wrap in a jQuery object.
+     * @param element A DOM element to wrap in a jQuery object.
      * @see \`{@link https://api.jquery.com/jQuery/ }\`
      * @since 1.0
      */
-    <T extends Element>(element_elementArray: T | ArrayLike<T>): JQuery<T>;
+    // tslint:disable-next-line:unified-signatures
+    <T extends Element>(element: T): JQuery<T>;
+    /**
+     * Return a collection of matched elements either found in the DOM based on passed argument(s) or created
+     * by passing an HTML string.
+     *
+     * @param elementArray An array containing a set of DOM elements to wrap in a jQuery object.
+     * @see \`{@link https://api.jquery.com/jQuery/ }\`
+     * @since 1.0
+     */
+    // tslint:disable-next-line:unified-signatures
+    <T extends Element>(elementArray: ArrayLike<T>): JQuery<T>;
     /**
      * Return a collection of matched elements either found in the DOM based on passed argument(s) or created
      * by passing an HTML string.
      *
      * @param selection An existing jQuery object to clone.
+     *                  An array not containing a set of DOM elements.
      * @see \`{@link https://api.jquery.com/jQuery/ }\`
      * @since 1.0
      */
-    <T>(selection: JQuery<T>): JQuery<T>;
+    <T>(selection: ArrayLike<T>): JQuery<T>;
```

Comment:
Are you sure `JQuery<T>` works for any type `T`? Can I have a `JQuery<number>`?
If so, this is exactly the same as the above except `extends Element` is missing -- couldn't the above be removed since we no longer require any constraint on `T`?

Reply:
It works, but it is not documented...
I added this method to make `$([[new HTMLParagraphElement()]])` work.
But, I'm wondering now if it was not a known limitation.

Indeed, though `jQuery.isPlainObject([[document.createElement("select")]])` is false,
```typescript
const t: JQuery.PlainObject<any> = [[document.createElement("select")]];
```
does compile while this should not.

So probably I should remove the method you marked. And put back `<T>(selection: JQuery<T>): JQuery<T>;`. This is anyway an small problem that can wait.